sdk/healthinsights/azure-healthinsights-radiologyinsights/README.md

```diff
@@ -7,7 +7,7 @@
 
 ### Prequisites
 
-- Python 3.8 or later is required to use this package.
+- [Python 3.8+][python] is required to use this package.
 - You need an [Azure subscription][azure_sub] to use this package.
 - An existing Cognitive Services Health Insights instance.
 
```

Comment:
> | SDK version | Supported API version of service |
> |-------------|----------------------------------|
> | 1.0.0b1     | 2023-09-01-preview               |

Above table needs update with row for GA version "1.0.0 | 2024-04-01".

Reply:
Updated